packages/node-rnw-rpc/windows/NodeRpc/Server.idl

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
are these going to be public APIs? if so can you add the comments as doc_string so we can generate docs from it?

Reply:
Will not be public as part of RNW docs. Likely just small package README.